PublicFolders/SourceSideValidations/SourceSideValidations.ps1

```diff
@@ -45,165 +45,170 @@ param (
 . $PSScriptRoot\..\..\Shared\Test-ScriptVersion.ps1
 . $PSScriptRoot\..\..\Shared\Out-Columns.ps1
 
-if (-not $SkipVersionCheck) {
-    if (Test-ScriptVersion -AutoUpdate) {
-        # Update was downloaded, so stop here.
-        Write-Host "Script was updated. Please rerun the command."
-        return
+try {
+    if (-not $SkipVersionCheck) {
+        if (Test-ScriptVersion -AutoUpdate) {
+            # Update was downloaded, so stop here.
+            Write-Host "Script was updated. Please rerun the command."
+            return
+        }
     }
-}
 
-$errorColor = "Red"
-$configuredErrorColor = (Get-Host).PrivateData.ErrorForegroundColor
-if ($configuredErrorColor -is [ConsoleColor]) {
-    $errorColor = $configuredErrorColor
-}
+    $errorColor = "Red"
+    $configuredErrorColor = (Get-Host).PrivateData.ErrorForegroundColor
+    if ($configuredErrorColor -is [ConsoleColor]) {
+        $errorColor = $configuredErrorColor
+    }
 
-$warningColor = "Yellow"
-$configuredWarningColor = (Get-Host).PrivateData.WarningForegroundColor
-if ($configuredWarningColor -is [ConsoleColor]) {
-    $warningColor = $configuredWarningColor
-}
+    $warningColor = "Yellow"
+    $configuredWarningColor = (Get-Host).PrivateData.WarningForegroundColor
```

Comment:
In stead of using this.... shouldn't we use the Write-Host from the shared location that does handle this?

Reply:
Those colors get passed to Out-Columns. I never pass them to Write-Host.